models.rb

```diff
@@ -122,6 +142,10 @@ class Level < ActiveRecord::Base
   has_many :scores, as: :highscoreable
   enum tab: [:SI, :S, :SU, :SL, :SS, :SS2]
 
+  def self.count
```

Comment:
I believe ActiveRecord has something like `Level.count` built in, so you shouldn't need to do this spooky `ObjectSpace` counting stuff :)